lib/npm/node-platform.ts

```diff
@@ -22,6 +22,7 @@ export const knownUnixlikePackages: Record<string, string> = {
   'linux arm LE': 'esbuild-linux-arm',
   'linux arm64 LE': 'esbuild-linux-arm64',
   'linux ia32 LE': 'esbuild-linux-32',
+  'linux riscv64 LE': 'esbuild-linux-riscv64',
```

Comment:
`riscv64` is not documented as something node ever returns for `process.arch`: https://nodejs.org/api/process.html#processarch:

> ## `process.arch`
>
> The operating system CPU architecture for which the Node.js binary was compiled. Possible values are: `'arm'`, `'arm64'`, `'ia32'`, `'mips'`,`'mipsel'`, `'ppc'`, `'ppc64'`, `'s390'`, `'s390x'`, `'x32'`, and `'x64'`.

I'm wondering if node's documentation is wrong because [this PR](https://github.com/nodejs/node/commit/1d21a8d140f78d4408c5375caedc917a83902a80#diff-4d5f3192809ec1b9add6b33007e0c50031ad9a0a2f3f55a481b506468824db2cR50) appears to indicate that node supports this architecture. I asked about that here: https://github.com/nodejs/node/issues/41900.